README.rst

```diff
@@ -121,17 +114,18 @@ Get information on fields and meta data
 **Get information on region ids**
 
 .. code-block:: python
-   :linenos:
 
+    # Version >= 0.1.2
     from datenguidepy import get_all_regions
+    # Version < 0.1.2
+    from datenguidepy.query_helper import get_all_regions
```

Comment:
Does also not appear to be a bug to me, with respect to the versioning. Still true if our next release is simply 0.2.0 though.